bedrock/styleguide/redirects.py

```diff
@@ -12,4 +12,8 @@
              'styleguide.websites.sandstone-intro'),
     redirect(r'^styleguide/websites/sandstone/tabzilla/?$',
              'styleguide.websites.sandstone-intro'),
+
+    # Bug 1329931
+    redirect(r'^styleguide/products/firefox-os(?:/.*)?$',
```

Comment:
Please change this line to `redirect(r'^styleguide/products/firefox-os(/.*)?$',`